changelog.yaml

```diff
@@ -51,12 +51,29 @@
 - version: NEXT
   date: TBD
   changes:
+
+    - type: bug
+      impact: patch
```

Comment:
I think we can have a minor here, as state is updated in a different way than before.

Reply:
done